caboose.go

```diff
@@ -85,11 +85,12 @@ const DefaultLoggingInterval = 5 * time.Second
 const DefaultSaturnOrchestratorRequestTimeout = 30 * time.Second
 
 const DefaultSaturnBlockRequestTimeout = 19 * time.Second
-const DefaultSaturnCarRequestTimeout = 30 * time.Minute
+
+const DefaultSaturnCarRequestTimeout = 1 * time.Minute
 const DefaultSaturnMirrorRequestTimeout = 30 * time.Second
 
 // default retries before failure unless overridden by MaxRetrievalAttempts
-const defaultMaxRetries = 3
+const defaultMaxRetries = 2
```

Comment:
~99% of ALL requests that succeed do so in the first two retries:

https://protocollabs.grafana.net/d/6g0_YjBVk/bifrost-caboose-staging?orgId=1&from=1692841797171&to=1692863397171&viewPanel=98

Reply:
this seems fine